conjur-openapi.yml

```diff
@@ -934,6 +934,86 @@ paths:
 
+  '/resources':
+    get:
+      tags:
+        - "resources"
+      summary: "Lists resources within an organization account."
+      description: "Lists resources within an organization account.
+
+
+      In the absence of an `account` query parameter, shows results for the account of the authorization token user.
+
+
+      If an `account` query parameter is given, shows results for the specified account.
+
+
+      If a `kind` query parameter is given, narrows results to only resources of that kind.
+
+
+      If a `limit` is given, returns no more than that number of results. Providing an `offset` skips a number of resources before returning the rest. In addition, providing an `offset` will give `limit` a default value of 10 if none other is provided. These two parameters can be combined to page through results.
+
+
+      If the parameter `count` is `true`, returns only the number of items in the list.
+
+
+      ##### Text search
+
+
+      If the `search` parameter is provided, narrows results to those pertaining to the search query. Search works across resource IDs and the values of annotations. It weighs results so that those with matching id or a matching value of an annotation called `name` appear first, then those with another matching annotation value, and finally those with a matching  `kind`."
+      operationId: "getResourcesQueryParameters"
+      parameters:
+      - name: "account"
+        in: "query"
+        description: "Organization account name"
+        example: myorg
+        schema:
+          $ref: '#/components/schemas/AccountName'
+      - name: "kind"
+        in: "query"
+        description: "Type of resource"
+        example: user
+        schema:
+          $ref: '#/components/schemas/Kind'
+      - name: "search"
+        in: "query"
+        description: "Filter resources based on this value by name"
+        example: "db"
+        schema:
+          $ref: '#/components/schemas/SearchFilter'
+      - name: "offset"
+        in: "query"
+        description: "When listing resources, start at this item number."
+        schema:
+          $ref: '#/components/schemas/Offset'
+      - name: "limit"
+        in: "query"
+        description: "When listing resources, return up to this many results."
+        schema:
+          $ref: '#/components/schemas/Limit'
+      - name: "count"
+        in: "query"
+        description: "When listing resources, if `true`, return only the count of the results."
+        schema:
+          $ref: '#/components/schemas/Count'
+
+      responses:
+        "200":
+          description: "The response body contains the list of matching resources"
+          content:
+            application/json:
+              schema:
+                type: object
+        "401":
+          $ref: '#/components/responses/UnauthorizedError'
+        "403":
+          $ref: '#/components/responses/InadequatePrivileges'
+        "500":
+          $ref: '#/components/responses/InternalServerError'
```

Comment:
I think we should have four test cases, checking that each of these four response codes, and any applicable response bodies, happen under the correct circumstances.

Reply:
Started working on this.
I assume this is applicable to all the other endpoints as well.
I'll file some issues.

Reply:
Yup - thank you!